introduction.md

```diff
@@ -1,2 +1,5 @@
 You just entered the dojo, here is a repo for some rules to know. Welcome.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?